recipes/r-scevan/meta.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: r-scevan
+  version: 1.0.1
+
+source:
+  url: https://github.com/AntonioDeFalco/SCEVAN/archive/refs/tags/v1.0.1.tar.gz  
+  sha256: 7558d31282c36418db3960b3edb9cb9815db14b0a48d8882287e11f752dfeab1       
+
+build:
+  number: 0
+  noarch: generic  # Specify that the package is noarch
+  run_exports: '{{ pin_compatible("r-scevan", max_pin="x.x") }}'
+  rpaths:
+    - lib/R/lib
+    - lib
+  script: "${R} CMD INSTALL --build . "
+
+requirements:
+  host:
+    - r-base
+    - r-remotes
+    - r-devtools
+    - r-rcpp
+  run:
+    - r-base
+    - r-devtools
+    - r-paralleldist
+    - r-pheatmap
+    - r-forcats
+    - r-dplyr
+    - bioconductor-fgsea
+    - r-cluster
+    - r-ggplot2
+    - r-Rtsne
+    - bioconductor-scran
+    - r-ape
+    - bioconductor-ggtree
+    - r-tidytree
+    - r-ggrepel
+    - r-optparse
+
+test:
+  commands:
+    - $R -e "library(devtools)"
+    - $R -e "devtools::install_github('miccec/yaGST')"
+    - $R -e "devtools::install_github('AntonioDeFalco/SCEVAN', ref = 'v1.0.1')"
```

Comment:
I suspect that these shouldn't be in the tests... yaGST needs to be installed in the script part.

Reply:
removed "install_github('AntonioDeFalco/SCEVAN')"; now installed with "R CMD INSTALL --build . "